code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1814,6 +1814,36 @@
 			H.facial_hair_style = "Beard (Very Long)"
 			H.update_hair()
 
+/datum/reagent/barbers_afro_mania
+	name = "Barber's Afro Mania"
+	description = "Unleash the funk within you."
+	reagent_state = LIQUID
+	color = "#FF8800"
+	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY | CHEMICAL_GOAL_BOTANIST_HARVEST
```

Comment:
`CHEMICAL_GOAL_BOTANIST_HARVEST` is wrong. It's not a chem that Botanist can harvest.

Reply:
Copy and pasting moment

Reply:
Done